Source/UIGraphBlock.cpp

```diff
@@ -20,7 +20,7 @@ namespace reverb
     * is updated periodically and on notify in a separate thread.
     */
     UIGraphBlock::UIGraphBlock(AudioProcessor& processor)
-        : processor(processor)
+        : processor(processor), juce::GroupComponent("IR Graph", "spectrum - impulse response")
```

Comment:
I'm not sure spectrum is currently the appropriate word given it isn't a frequency response of the IR.  I do know you were simply following the Photoshop. I can't think of an alternative word maybe @segwin can help us out with that.  Also, maybe we should capitalize 1st letter of every word for outline titles (suggestion).

Reply:
Maybe "waveform - impulse response"?

As an aside, aesthetically I like the lowercase titles for sections but that's obviously a matter of taste.